sonar-plugin-api/src/main/java/org/sonar/api/CoreProperties.java

```diff
@@ -328,7 +328,9 @@
 
   /**
    * @since 3.3
+   * @deprecated since 7.1, developer connection link is no more feed
    */
+  @Deprecated
   String LINKS_SOURCES_DEV = "sonar.links.scm_dev";
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Do not forget to remove this deprecated code someday. ([squid:S1133](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1133))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.sonarqube%3Asonarqube&branch=feature%2Fstas%2Fproject-links&issues=AWGTablbQeDOAQ7I7d6-&open=AWGTablbQeDOAQ7I7d6-)

[ISSUE_KEY]: AWGTablbQeDOAQ7I7d6-